docs/overview/06-verification_pallets/02-zksync_era.md

```diff
@@ -1,15 +1,44 @@
 ---
-title: zkSync Era Verifier
+title: ZkSync Era Verifier
 ---
 
-## [`settlementZksyncPallet`](https://github.com/HorizenLabs/NH-core/tree/main/pallets/settlement-zksync)
+## [`settlementZksyncPallet`](https://github.com/HorizenLabs/NH-core/tree/main/verifiers/zksync)
 
-The [`submitProof`](https://github.com/HorizenLabs/NH-core/tree/main/pallets/settlement-zksync/src/lib.rs#L103)
-extrinsic can be used to verify ZkSync-Era proofs; these proofs are generated by Boojum prover that's a
+### Statement hash components
+
+- context: `keccak256(b"zksync")`
+- vk: `keccak256([])`
+- pubs: `keccak256(pubs)`
+
+### `Verifier` implementation
+
+This `Verifier` implementation can be used to verify ZkSync-Era proofs; these proofs are generated by Boojum prover that's a
 STARK-based proof system. ZkSync-Era roolups generate these proofs and you can verify them on ZkVerify instead of
-post them directly on Ethereum. The input proof format is 1440 bytes array where the last 32 bytes are the
+post them directly on Ethereum.
 public inputs: the pallet use [`zksync-era-verifier` crate](https://github.com/HorizenLabs/zksync-era-verifier/tree/v0.1.0)
-to deserialize the proof and public inputs and then verify them. The pallet's duties are summarized in the following code snippet:
+
+- `verify_proof()` uses [`zksync-era-verifier` crate](https://github.com/HorizenLabs/zksync-era-verifier/tree/v0.1.0) to deserialize
+the proof and then verify them against the given public inputs.
+- Define the following types:
+
+    ```rust
+    pub type Pubs = [u8; 32];
+    pub type Proof = [u8; 1402];
+    pub type Vk = ();
+    ```
+
+- hash context data is `b"zksync"`
+- the pubs bytes are the input ones
+
+#### Note
+
+In this pallet doesn't have any sense register the verification key before
+to use it in `submitProof` exstrinsic because the only valid one is `()`:
+the void key.
```

Comment:
```suggestion
In this pallet it doesn't make sense to register any verification key, because the only valid one is `()`: the void key.
```
Let's also mention briefly why